checker/check.ml

```diff
@@ -353,7 +353,10 @@ let intern_from_file ~intern_mode (dir, f) =
       let opaque_csts = marshal_in_segment ~validate ~value:Values.v_univopaques ~segment:seg_univs f ch in
       let tasks = marshal_in_segment ~validate ~value:Values.(Opt Any) ~segment:seg_tasks f ch in
       let table = marshal_in_segment ~validate ~value:Values.v_opaquetable ~segment:seg_opaque f ch in
-      let vmlib = marshal_in_segment ~validate ~value:Values.v_vmlib ~segment:seg_vmlib f ch in
+      let vmlib = if enable_VM
+        then marshal_in_segment ~validate ~value:Values.v_vmlib ~segment:seg_vmlib f ch
+        else Vmlibrary.(export (set_path dir empty))
```

Comment:
I think that rather than a dummy value, we should change the library_vm field to an option.